### PR TITLE
Add warning for Baremetal and vSphere registry storage

### PR DESCRIPTION
**Pull request by bmcelvee**
This came up in a slack conversation with Ali. 

FYI - @aheslin @sferich888 

We are still on merge freeze, so I will merge on Monday.

### CODE REVIEW

**Comment by sferich888** [comment]
Can we just link to the docs section that defined the best practices for storage, not add a note that has to be maintained?

**Comment by bmcelvee** [comment]
Thanks! I added them as additional resources links. 

@openshift/team-documentation PTAL, this one is tiny.

File: modules/registry-configuring-storage-baremetal.adoc [hunk 1]
```diff
@@ -22,13 +22,13 @@ registry to use storage.
...
 . Verify you do not have a registry pod:
 +
 ----
 $ oc get pod -n openshift-image-registry
 ----
-
++
 [NOTE]
 =====
 If the storage type is `emptyDIR`, the replica number can not be greater than `1`.
```
Comment (bobfuru) [inline modules/registry-configuring-storage-baremetal.adoc]: s/can not/cannot

File: modules/registry-configuring-storage-vsphere.adoc [hunk 2]
```diff
@@ -22,19 +22,20 @@ registry to use storage.
...
++
 * Must have "100Gi" capacity.
 
 .Procedure
 
 . To configure your registry to use storage, change the `spec.storage.pvc` in the
 `configs.imageregistry/cluster` resource.
-
++
 . Verify you do not have a registry pod:
```
Comment (bobfuru) [inline modules/registry-configuring-storage-vsphere.adoc]: s/pod/Pod ?
Elsewhere in the docs (such as Storage), and in the [Documentation guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc), Pod is capitalized.

**Review by bobfuru — COMMENTED** [review]
Couple minor items; otherwise, LGTM!

**Comment by bmcelvee** [comment]
Thanks, @bobfuru!

**Comment by bmcelvee** [comment]
This PR is adding a link in the additional resources section of two assemblies and making minor edits. No QE review required.

**Comment by bmcelvee** [comment]
I think I'm going to need to open a separate PR to make the changes on 4.1 since the file structure changed. I can do that tomorrow.

**Pull request merged**